src/game/logic/object/update/aiupdate.h

```diff
@@ -56,21 +68,51 @@ enum LocomotorSetType
     LOCOMOTORSET_COUNT,
 };
 
+enum MoodMatrixAction
+{
+    MM_ACTION_IDLE = 0,

```

Comment:
0 isnt needed if thats where we start

Reply:
I think it is fine to be explicit. It also makes it clear that it is not an oversight.

Reply:
fixed.